packages/SwingSet/package.json

```diff
@@ -37,6 +37,7 @@
     "@agoric/harden": "^0.0.4",
     "@agoric/marshal": "^0.1.1",
     "@agoric/nat": "^2.0.1",
+    "@agoric/simple-swing-store": "^0.1.0",
```

Comment:
don't wanna bikeshed too much, but maybe `swing-store-simple` or `swing-store-memory`, so all flavors will sort next to `swingset`?

Reply:
Yes, that totally makes sense.  I'll put that into this PR shortly.